yt_dlp/extractor/flextv.py

```diff
@@ -0,0 +1,50 @@
+from .common import InfoExtractor
+from ..networking.exceptions import HTTPError
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    parse_iso8601,
+    str_or_none,
+    traverse_obj,
+    url_or_none,
+    UserNotLive,
+)
+
+
+class FlexTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?flextv\.co\.kr/channels/(?P<id>\d+)/live'
+    _TESTS = [{
+        'url': 'https://www.flextv.co.kr/channels/570421/live',
+        'only_matching': True,
+    }, {
```

Comment:
add an actual test, you can use regex for the title, and `int` for the timestamp if needed

Reply:
That test should be skipped, right?

Reply:
if it's not a 24/7 stream, then yes